src/components/Layout/index.tsx

```diff
@@ -8,7 +10,35 @@ type Props = {
 function Layout(props: Props) {
   const { children } = props;
 
-  return <div className={styles.layout}>{children}<Footer /></div>;
+  const navBarContent = (title: string, refUrl: string) => {
+    const router =useRouter()
+
+    const linkClasses = `link ${router.pathname === refUrl ? styles.active : ''}`
+    
+    return (
+      <Link href={refUrl}>
+        <a className={linkClasses}>{title}</a>
+      </Link>
+    );
+  };
```

Comment:
We can move this function outside of `Layout` because every time the `Layout` component renders, it'll reinitialize the `navBarContent` function.

## Probable Issue
- We won't be able to use the `useRouter` hook in the `navBarContent` function after moving it out. (Hooks only work in Functional Components)

## Solution
- You can pass the variable `router` as a parameter to `navBarContent` function

Reply:
Thanks for the review and solution! 👍 
Done with requested changes! 😄